config/test.exs

```diff
@@ -24,7 +24,7 @@ config :tesla, adapter: Tesla.Mock
 config :omscore, Omscore.Repo,
   adapter: Ecto.Adapters.Postgres,
   username: "postgres",
-  password: "postgres",
+  password: "5ecr3t",
```

Comment:
i think this should be put back to postgres to not break what's in prod

Reply:
it's testing, it shouldn't. I didn't touch any other files and this is just to make it consistent.

Reply:
I can delete it, but for prod it will make no difference as the only place testing env is used is CI

Reply:
true